code/modules/mob/living/simple_animal/hostile/true_changeling.dm

```diff
@@ -0,0 +1,193 @@
+	amount *= 2 //True changelings take 2 times the burn damage!
+	..()
+
+/mob/living/simple_animal/hostile/true_changeling/verb/reform()
+	set name = "Re-Form Human Shell"
+	set desc = "We turn back into a human. This takes considerable effort and will stun us for some time afterwards."
+	set category = "True Changeling"
+
+	if(stat)
+		to_chat(usr, "<span class='warning'>We must be conscious to debase ourselves!</span>")
+		return 0
+	if(!stored_changeling)
+		to_chat(usr, "<span class='warning'>We do not have a form other than this!</span>")
+		return 0
+	if(stored_changeling.stat == DEAD)
+		to_chat(usr, "<span class='warning'>Our human form is dead!</span>")
+		return 0
+	if(time_spent_as_true < TRUE_CHANGELING_REFORM_THRESHOLD)
+		to_chat(usr, "<span class='warning'>We are not able to change back at will!</span>")
+		return 0
+	visible_message("<span class='warning'>[src] suddenly crunches and twists into a smaller form!</span>", \
+					"<span class='danger'>We return to our lesser form.</span>")
+	stored_changeling.loc = get_turf(src)
+	mind.transfer_to(stored_changeling)
+	stored_changeling.Unconscious(80)
+	stored_changeling.status_flags &= ~GODMODE
+	qdel(usr)
+	return 1
+
+/mob/living/simple_animal/hostile/true_changeling/verb/devour()
+	set name = "Devour"
+	set desc = "We tear into the innards of a human. After some time, they will be significantly damaged and our health partially restored."
+	set category = "True Changeling"
+
+	if(stat)
+		to_chat(usr, "<span class='warning'>We must be conscious to feast!</span>")
+		return 0
+	if(devouring)
+		to_chat(usr, "<span class='warning'>We are already feasting on a human!</span>")
+		return 0
+	var/list/potential_targets = list()
+	for(var/mob/living/carbon/human/H in range(1, src))
+		if(H == stored_changeling || (H.mind && H.mind.changeling)) //You can't eat changelings in human form
+			continue
+		potential_targets.Add(H)
+	if(!potential_targets.len)
+		to_chat(usr, "<span class='warning'>There are no humans nearby!</span>")
+		return 0
+	var/mob/living/carbon/human/lunch
+	if(potential_targets.len == 1)
+		lunch = potential_targets[1]
+	else
+		lunch = input(src, "Choose a human to devour.", "Lunch") as null|anything in potential_targets
+	if(!lunch)
+		return 0
+	if(lunch.getBruteLoss() >= 200)
+		to_chat(usr, "<span class='warning'>This human's flesh is too mangled to devour!</span>")
+		return 0
+	devouring = TRUE
+	visible_message("<span class='warning'>[src] begins ripping apart and feasting on [lunch]!</span>", \
+						"<span class='danger'>We begin to feast upon [lunch]...</span>")
+	if(!do_mob(src, 50, target = lunch))
+		devouring = FALSE
+		return 0
+	devouring = FALSE
+	visible_message("<span class='warning'>[src] tears a chunk from [lunch]'s flesh!</span>", \
+						"<span class='danger'>We tear a chunk of flesh from [lunch] and devour it!</span>")
+	lunch.adjustBruteLoss(60)
+	lunch << "<span class='userdager'>[src] tears into you!</span>"
```

Comment:
How come you managed to_chat in half the file but not here

Reply:
i missed one

Reply:
"<span class='userdager' 🤔 🤔 🤔 🤔